crates/debug/src/transform/expression.rs

```diff
@@ -7,14 +7,28 @@ use wasmtime_environ::entity::EntityRef;
 use wasmtime_environ::ir::{StackSlots, ValueLabel, ValueLabelsRanges, ValueLoc};
 use wasmtime_environ::isa::RegUnit;
 use wasmtime_environ::wasm::{get_vmctx_value_label, DefinedFuncIndex};
+use wasmtime_environ::ModuleMemoryOffset;
 
 #[derive(Debug)]
 pub struct FunctionFrameInfo<'a> {
     pub value_ranges: &'a ValueLabelsRanges,
-    pub memory_offset: i64,
+    pub memory_offset: ModuleMemoryOffset,
     pub stack_slots: &'a StackSlots,
 }
 
+impl<'a> FunctionFrameInfo<'a> {
+    fn vmctx_memory_offset(&self) -> Option<i64> {
+        match self.memory_offset {
+            ModuleMemoryOffset::Defined(x) => Some(x as i64),
+            ModuleMemoryOffset::Imported(_) => {
+                // TODO implement memory offset for imported memory
```

Comment:
File an issue so this doesn't get forgotten?

Reply:
https://github.com/bytecodealliance/wasmtime/issues/896